package.json

```diff
@@ -27,10 +27,11 @@
     "hammerjs": "^2.0.8",
     "rxjs": "^6.5.5",
     "rxjs-compat": "^6.5.5",
+    "transbank-sdk": "^2.1.3",
```

Comment:
This is not place to update the library, you can see "server" folders.

Reply:
ups sorry will take a look at this

Reply:
Do I have to only update / add it in the server/package.json?